pxd_fastpath.c

```diff
@@ -416,6 +416,45 @@ static void pxd_failover_initiate(struct pxd_device *pxd_dev, struct pxd_io_trac
 	queue_work(pxd_dev->fp.wq, &head->wi);
 }
 
+static int remap_io_status(int status)
```

Comment:
io status reconciliation from all replicas. Not all failures can be failedover... exceptions below, rest are converted to a hard -EIO error which triggers a failover, if allowed.